Casks/slack.rb

```diff
@@ -1,21 +1,32 @@
 cask "slack" do
   arch arm: "arm64", intel: "x64"
 
-  version "4.31.152"
-  sha256 arm:   "ec227b3181732c554b1e2652f05573875a950256d30218344ccd9445a91ad0d6",
-         intel: "62e0c4d266de7f460769bdbafce1726718061fb72826bd8d7cd1f3470ab38132"
+  on_mojave do
```

Comment:
```suggestion
  on_mojave :or_older do
```

Will ensure the cask can be read on older system versions, but still error on the `depends_on` as expected.